test/Microsoft.OpenAPI.OData.Reader.Tests/Resources/TripService.OpenApi.V2.json

```diff
@@ -28511,12 +28511,34 @@
     },
     "Microsoft.OData.Service.Sample.TrippinInMemory.Models.PersonGender": {
       "title": "PersonGender",
+      "description": "Gender of the person.",
       "enum": [
```

Comment:
can we extend the item in the array to be a complex object,  which has the "name" and description? 

In this case, we don't need to add an extension to repeat the name or value for enum member?

Reply:
here I'm just implementing this spec so people also get compatibility with auto rest should they be using it afterwards https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-enum